{
   "active_lock_reason" : "resolved",
   "assignee" : null,
   "assignees" : [],
   "author_association" : "NONE",
   "body" : "I have installed core 0.14.0rc1 after running my node (Raspberry Pi 2) successfully on 0.13.2 for weeks .\r\n\r\nAfter upgrading I am getting errors and having issues keeping the node running. Once in a while core crashes with this error in the terminal:\r\n\r\n************************\r\nEXCEPTION: St9bad_alloc       \r\nstd::bad_alloc       \r\nbitcoin in scheduler       \r\n\r\nterminate called after throwing an instance of 'std::bad_alloc'\r\n  what():  std::bad_alloc\r\n\r\nThis happens when I use previously downloaded blockchain and also when I start to download the whole blockchain all over again.\r\n\r\nMy system settings:\r\nRaspberry Pi 2\r\n4GB SD card\r\nNFS share to Synology NAS (blockchain is successfully stored on the nas). Have a backup on an 128GB USB Stick in the pi.",
   "closed_at" : "2017-10-08T03:37:16Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/863730?v=4",
      "events_url" : "https://api.github.com/users/fanquake/events{/privacy}",
      "followers_url" : "https://api.github.com/users/fanquake/followers",
      "following_url" : "https://api.github.com/users/fanquake/following{/other_user}",
      "gists_url" : "https://api.github.com/users/fanquake/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/fanquake",
      "id" : 863730,
      "login" : "fanquake",
      "node_id" : "MDQ6VXNlcjg2MzczMA==",
      "organizations_url" : "https://api.github.com/users/fanquake/orgs",
      "received_events_url" : "https://api.github.com/users/fanquake/received_events",
      "repos_url" : "https://api.github.com/users/fanquake/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/fanquake/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/fanquake"
   },
   "comments" : 24,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9857/comments",
   "created_at" : "2017-02-25T08:15:53Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9857/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/9857",
   "id" : 210220297,
   "labels" : [
      {
         "color" : "981023",
         "default" : false,
         "description" : null,
         "id" : 326918230,
         "name" : "Resource usage",
         "node_id" : "MDU6TGFiZWwzMjY5MTgyMzA=",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Resource%20usage"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9857/labels{/name}",
   "locked" : true,
   "milestone" : null,
   "node_id" : "MDU6SXNzdWUyMTAyMjAyOTc=",
   "number" : 9857,
   "performed_via_github_app" : null,
   "reactions" : {
      "+1" : 1,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 1,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9857/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "state_reason" : "completed",
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9857/timeline",
   "title" : "Sync problem crashes Bitcoin: EXCEPTION: St9bad_alloc",
   "updated_at" : "2022-02-15T11:03:38Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9857",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/1825953?v=4",
      "events_url" : "https://api.github.com/users/Smiggel/events{/privacy}",
      "followers_url" : "https://api.github.com/users/Smiggel/followers",
      "following_url" : "https://api.github.com/users/Smiggel/following{/other_user}",
      "gists_url" : "https://api.github.com/users/Smiggel/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/Smiggel",
      "id" : 1825953,
      "login" : "Smiggel",
      "node_id" : "MDQ6VXNlcjE4MjU5NTM=",
      "organizations_url" : "https://api.github.com/users/Smiggel/orgs",
      "received_events_url" : "https://api.github.com/users/Smiggel/received_events",
      "repos_url" : "https://api.github.com/users/Smiggel/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/Smiggel/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/Smiggel/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/Smiggel"
   }
}
